internal/provisioner/plain/README.md

```diff
@@ -123,57 +127,44 @@ Once the cluster has been setup, take the following steps:
 From there, create some `Bundles` and `BundleInstance` types to see the provisioner in action. For an example bundle to
 use, the [combo operator](https://github.com/operator-framework/combo) is a good candidate.
 
-Create the bundle:
+Create the combo `BundleInstance` referencing the desired combo `Bundle` configuration.
 
 ```console
 $ kubectl apply -f -<<EOF
 apiVersion: core.rukpak.io/v1alpha1
-kind: Bundle
+kind: BundleInstance
 metadata:
-  name: combo-v0.0.1
+  name: combo
 spec:
-  source:
-    type: image
-    image:
-      ref: quay.io/tflannag/bundle:combo-operator-v0.0.1
   provisionerClassName: core.rukpak.io/plain
+  template:
+    metadata:
+      labels:
+        subscription: combo
```

Comment:
nit: people will likely copy this pattern if its in our docs. curious what the thought process is behind the use of "subscription" as a term here. maybe "package" or even more generic, "app", instead?